src/plugins/chinese/linovelib_cn.ts

```diff
@@ -4,9 +4,9 @@ import { FilterTypes, Filters } from '@libs/filterInputs';
 import { Plugin } from '@typings/plugin';
 import { NovelStatus } from '@libs/novelStatus';
 
-class Linovelib implements Plugin.PluginBase {
-  id = 'linovelib';
-  name = 'Linovelib';
+class Linovelib_cn implements Plugin.PluginBase {
+  id = 'linovelib-cn';
+  name = 'Linovelib-cn';
```

Comment:
You should keep the old id, so people can still update on it instead of downloading the a new plugin.
I sugguest the name format `Linovelib (繁體)`. It would be useful for Chinese reader to recognize

Reply:
Ok, I will change it later

Reply:
this was solved in the new commit